example/README.md

```diff
@@ -0,0 +1,68 @@
+# UserALE.js Example
+
+This directory provides 
+ * example usage for UserALE.js within a simple HTML Webpage,
+ * how users can expect to log UserALE events, and
+ * what log events looks like within a simple UserALE logging server.
+
+## Prerequisites
+
+Please see the [Prerequsites](https://github.com/apache/incubator-senssoft-useralejs#prerequsites)
+
+## Running the Logging Server
+
+Assuming you have [node.js](https://nodejs.org/) installed, simply run
+
+```
+node server.js
```

Comment:
Should just be `npm run example:watch`

Reply:
@rgfoley thanks please scope updated PR, thanks